ignite/contrib/metrics/regression/canberra_metric.py

```diff
@@ -36,7 +36,23 @@ class CanberraMetric(_BaseRegression):
 
     .. _`Botchkarev 2018`:
         https://arxiv.org/ftp/arxiv/papers/1809/1809.03006.pdf
-
+    
+    .. testcode::
+
+            def process_function(engine, batch):
+                y_pred, y = batch
+                return y_pred, y
+            engine = Engine(process_function)
+            metric = CanberraMetric()
+            metric.attach(engine, 'canberra')
+            preds = torch.Tensor([[3.8], [9.9], [-5.4], [2.1]])
```

Comment:
I’m wondering whether `y_true` is a better name than `preds`.

```suggestion
            y_true = torch.Tensor([[3.8], [9.9], [-5.4], [2.1]])
```
and `targets` would be `y_pred`.

What do you think ? The point is we have to rename a plenty of doctests…